documentation/src/main/asciidoc/topics/proc_configuring_cluster_role_mapper.adoc

```diff
@@ -1,27 +1,37 @@
 [id='configuring-cluster-role-mapper_{context}']
-= Configuring the cluster role mapper
+= Configuring the cluster role and permission mappers
 
-The cluster role mapper maintains a dynamic mapping between principals and roles, by storing it in the cluster registry.
-You can manipulate these mappings using either the CLI or the REST API.
-By default, the mapping is performed using a strict string equivalence between principal names and roles. It is possible to configure the cluster role mapper to apply transformation to the principal name before performing a lookup.
+The cluster role mapper maintains a dynamic mapping between principals and roles.
+The cluster permission mapper maintains a dynamic set of role definitions.
```

Comment:
From what I tested the permission mapper will have the same dynamic mapping as role-mapper, right?

Reply:
It's not possible to manipulate roles unless a MutableRolePermissionMapper is available (such as the ClusterPermissionMapper)

Reply:
ok, thats new and maybe not merged.
But a user might change the configuration later and then the stored roles or role-mappings are ignored

Reply:
Not new: https://github.com/infinispan/infinispan/blob/main/server/rest/src/main/java/org/infinispan/rest/resources/SecurityResource.java#L120 and https://github.com/infinispan/infinispan/blob/main/server/rest/src/main/java/org/infinispan/rest/resources/SecurityResource.java#L69

Reply:
Attempting to create a role with a RolePermissionMapper that does not implement MutableRolePermissionMapper will result in a `409 CONFLICT`